test/FIRParser/basic.fir

```diff
@@ -78,13 +79,13 @@ circuit MyModule :     ; CHECK: firrtl.circuit "MyModule" {
     ; CHECK: %_t = firrtl.wire {name = "_t"} : !firrtl.vector<uint<1>, 12>
     wire _t : UInt<1>[12] @[Nodes.scala 370:76]
 
-    ; CHECK: %_t_2 = firrtl.wire {name = "_t_2"} : !firrtl.vector<uint<1>, 13>
-    wire _t_2 : UInt<1>[13]
```

Comment:
Since `_t_2` is connected to `_t`, they have to have the same number of elements. I'm assuming it was intended for them to be valid to connect, so I've updated `_t_2` to match the number of elements in `_t`.

Reply:
Yeah I suspect that there are a lot of hand written tests that are buggy (because I don't have all the rules internalized).  Please feel free to adjust them as you encounter them. Thanks!